test/e2e/util_test.go

```diff
@@ -80,9 +81,11 @@ func buildEchoPod(name, namespace string) *corev1.Pod {
 
 func waitForHTTPClientCondition(t *testing.T, httpClient *http.Client, req *http.Request, interval, timeout time.Duration, compareFunc func(*http.Response) bool) error {
 	t.Helper()
+	req.Close = true
```

Comment:
This forces/changes the behaviour of this utility function. Let the caller decide and, if necessary, the caller can set the desired behaviour.

Reply:
Ack - removing as I was using it to test with and don't believe it is needed.